modules/Core/classes/Core/Notification.php

```diff
@@ -76,25 +104,31 @@ public function send(): void {
             )->first();
 
             if ($preferences->alert) {
-                $this->sendAlert($id, $content);
+                $this->sendAlert($id, $title, $content);
             }
             if ($preferences->email) {
-                $this->sendEmail($id, $content);
+                $this->sendEmail($id, $title, $content);
             }
         }
     }
 
-    private function sendAlert(int $userId, string $content): void {
-        Alert::send($userId, $this->_title, $content, null, $this->_skipPurify);
+    private function sendAlert(int $userId, string $title, ?string $content): void {
+        Alert::send(
+            $userId,
+            $title,
+            $this->_alertUrl ? null : $content,
```

Comment:
Why should content be null when alert url is set?

Reply:
AFAIK, alerts will ignore content if the url/link is set, so no need to set content if we set the URL instead